src/pages/workspaces/workspace/Data.js

```diff
@@ -1002,6 +1047,26 @@ const WorkspaceData = _.flow(
               }
             }, [`${type} (${typeDetails.count})`])
           }, sortedEntityPairs),
+          div({ style: Style.navList.heading }, ['Snapshots']),
+          _.map(([snapshotName, { id: snapshotId, entityMetadata: snapshotTables }]) => {
+            const snapshotTablePairs = _.flow(_.toPairs, _.sortBy(_.first))(snapshotTables)
+            return h(SnapshotCollapse, {
+              buttonStyle: { color: colors.dark(), fontWeight: 600 },
+              style: { fontSize: 14, marginRight: '0.5rem', lineHeight: '50px', padding: '0 1.5rem', borderBottom: `1px solid ${colors.dark(0.2)}` },
+              title: snapshotName
+            }, [
+              p({ style: { fontSize: 14, lineHeight: '1.5' } },
+                _.map(([tableName, { count }]) => {
+                  return [h(DataTypeButton, {
```

Comment:
This shouldn't return a one-member array, it should just be the `DataTypeButton`. Note that these buttons have keys on them, so that they can be members of the array from this `map` without anything becoming unhappy.

Stylistically, are we ok with the selection indicator being indented from the right margin? It looks a little weird to my eye; I think I'd prefer to see it indented from the left but not the right.

Reply:
the selection indicator has been fixed!